jenkins/tests/test_job.py

```diff
@@ -1,8 +1,8 @@
-from __future__ import absolute_import
+# pylint: disable=missing-module-docstring
+from unittest import TestCase
 
 from mock import patch
```

Comment:
Do you want to handle the switch to `unittest.mock` in this PR or do that separately?

Reply:
That will be done with Python 3.8 compatibility.